asyncscope.md

```diff
@@ -2317,7 +2317,236 @@ spec here
 
 ## `execution::simple_counting_scope`
 
-spec here
+Add the following new section imediate after **TODO**:
+
+::: add
+### Scopes [exec.scopes]
+
+#### Simple Counting Scope [exec.simple.counting.scope]
+
+##### General [exec.scounting.general]
+
+```
+class simple_counting_scope {
```

Comment:
What's the Committee's policy on using `struct` or `class`? In section 5, this is a `struct`, but that's basically just my whim; if `class` is expected, we should probably update section 5 to match.

Reply:
I think entities which are bags of data like `pair` or customisation objects would be `struct`s. Things with `private` members would be `class`. That's easily changed anyway.